arviz/tests/test_data_pyro.py

```diff
@@ -48,6 +50,9 @@ def test_inference_data(self, data, eight_schools_params):
             "prior": ["mu", "tau", "eta"],
             "prior_predictive": ["obs"],
         }
+        if packaging.version.parse(pyro.__version__) >= packaging.version.parse("1.0.0"):

```

Comment:
@fehiepsi Thanks for making change. 
I would still ask that this if statement is split out into another test sort of like this below. This way its more obvious that portions of test are being skipped if someone tests on old env.

`@pytest.mark.skipif(packaging.version.parse(pyro.__version__) >= packaging.version.parse("1.0.0"), reason="requires pyro 2.0.0 or higher")
def test_inference_data(self, data, eight_schools_params):
        inference_data = self.get_inference_data(data, eight_schools_params)
        test_dict = {
            "posterior": ["sample_stats"],
            "sample_stats": ["observed_data"],
            }
```
`

Reply:
It is fine for me to split the tests. I'll do that.